### ISSUE TITLE
why is latest inMobi minSDK 24 ?

### ISSUE THREAD

**Issue by fawxyz** [googleads/googleads-mobile-android-mediation#534]
in the latest version of inmobi adapter
com.google.ads.mediation:inmobi:10.7.7.0
the minSDK is 24 , is this intentional?

the release note doesn't state that minSDK changed

the version before, com.google.ads.mediation:inmobi:10.7.5.0 is minSDK 21

**Comment by matteinn** [comment]
I have the very same concern.
minSDK has been bumped to 24 together with the 10.7.5 -> 10.7.7 SDK bump via https://github.com/googleads/googleads-mobile-android-mediation/commit/ae94f7ac7c274bb5e507e2cf2b20668a1afd1806#diff-a7296b61bd15bfa53b79312b2f1803b5e1eca894e3ae744874b5ca7c9873661fR23

I reached out to InMobi and they confirm the new SDK 10.7.7 still supports Android 5 (API level 21).

Please clarify why the min SDK has been bumped and/or revert that change.

**Comment by fawxyz** [comment]
@matteinn 
looks like Google updated the release note, i checked https://developers.google.com/admob/android/mediation/inmobi#inmobi-android-mediation-adapter-changelog

Version 10.7.7.0
Verified compatibility with InMobi Kotlin SDK version 10.7.7.
Updated the minimum required Android API level to 24.

are you sure InMobi confirm that the latest sdk support API level 21?

**Comment by matteinn** [comment]
@fawxyz yes they confirmed API level 21 is still supported. I think the next adapter version will revert that unintended minSdk bump: that's something in charge of the google team, not InMobi.

**Comment by matteinn** [comment]
Hey @fawxyz, they are working on reverting the change. The next adapter version will support Android 5.

https://github.com/googleads/googleads-mobile-android-mediation/commit/cbd14f458944decb95b74d7feb33c855578397ac

**Comment by fawxyz** [comment]
@matteinn it's out already. And i already did update to
    implementation 'com.google.ads.mediation:inmobi:10.7.8.0'
but it won't run, still api 24

`Manifest merger failed : uses-sdk:minSdkVersion 21 cannot be smaller than version 24 declared in library [com.google.ads.mediation:inmobi:10.7.8.0]`

i guess they forgot to change some things.. or didn't test the release..

**Comment by matteinn** [comment]
@fawxyz 10.7.8.0 still requires 24, the next one will require 21 😉

**Comment by fawxyz** [comment]
@matteinn hopefully. but i still wonder if this is intentional or just error in their side. like, they already made error with 10.7.7.0 , and now 10.7.8.0 doesn't resolve it.

**Comment by matteinn** [comment]
@fawxyz this should be fixed in version 10.7.8.1, I didn't test but seems to be ok based on their commit https://github.com/googleads/googleads-mobile-android-mediation/commit/c8d48b99a01b6b248802e44cf43d9776cb03ab57

**Comment by fawxyz** [comment]
@matteinn I updated it last night. minSDK has returned to 21 and it ran. So it seems OK.
